src/Symfony/Component/EventDispatcher/EventDispatcher.php

```diff
@@ -139,7 +139,7 @@ public function hasListeners($eventName = null)
     /**
      * {@inheritdoc}
      */
-    public function addListener($eventName, $listener, $priority = 0)
+    public function addListener(string $eventName, $listener, int $priority = 0)
```

Comment:
Note: I intentionally did not add `callable $listener` because this would cause a call like…
```php
$dispatcher->addListener('some_event', ['Some\Class', 'someMethod']);
```
… to trigger autoloading for the class name that is passed here. This might have performance implications if we're adding a lot of static listeners.